ansible_galaxy/actions/install.py

```diff
@@ -421,7 +433,7 @@ def install_repository(galaxy_context,
                                                  display_callback=display_callback)
     except exceptions.GalaxyError as e:
         msg = "- %s was NOT installed successfully: %s "
-        display_callback(msg % (found_repository_spec.label, e), level='warning')
+        display_callback(msg % (found_repository_spec, e), level='warning')
         log.warning(msg, found_repository_spec.label, str(e))
         raise_without_ignore(ignore_errors, e)
```

Comment:
If this is an error, which it is because we terminate here without `--ignore-errors`, then it should be logged and displayed as an error, and not a warning.

Reply:
Think I'm going to leave it for now, and revisit when I get rid of raise_without_ignore().